firebase-firestore/src/androidMain/kotlin/dev/gitlive/firebase/firestore/GeoPoint.kt

```diff
@@ -0,0 +1,19 @@
+package dev.gitlive.firebase.firestore
+
+import kotlinx.serialization.Serializable
+
+/** A class representing a platform specific Firebase GeoPoint. */
+actual typealias NativeGeoPoint = com.google.firebase.firestore.GeoPoint
+
+/** A class representing a Firebase GeoPoint. */
+@Serializable(with = GeoPointSerializer::class)
```

Comment:
this approach is similar to a PR [#312](https://github.com/GitLiveApp/firebase-kotlin-sdk/pull/312)